inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -21,7 +21,7 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*(QuantGroupConv2D).*)",
         R"(.*(QuantGroupConv3D).*)",
         // TODO: Issue 31845
-        R"(.*(FakeQuantize).*)",
+        R"(.*(FakeQuantizeLayerTest).*)",
```

Comment:
Since FQ and ScSh fusion is now fixed I believe FQ single layer test should work correctly. Could you please try to remove this line?

Reply:
12 tests still fail, listed below: FakeQuantize/FakeQuantizeLayerTest...